devito/builtins.py

```diff
@@ -205,6 +207,9 @@ def mmin(f):
         Input operand.
     """
     if not isinstance(f, dv.Function):
+        if dv.configuration['mpi']:
+            warning("returning local min of array,"
```

Comment:
As I wrote on Slack, I would actually be even more restrictive 

```
if not isinstance(f, dv.Function):
    raise ValueError("Expected Function, not `%s`" % type(f))
...
```

that is, I'd raise a ValueError regardless of MPI